server/base/src/main/java/org/apache/accumulo/server/compaction/FileCompactor.java

```diff
@@ -429,7 +429,13 @@ private void compactLocalityGroup(String lgName, Set<ByteSequence> columnFamilie
             } else {
               metrics.incrementMajCPause();
             }
-            Thread.sleep(500);
+            try {
+              Thread.sleep(500);
+            } catch (InterruptedException e) {
+              Thread.interrupted();
```

Comment:
Why `Thread.interrupted();` to clear the interrupt status flag?  I think just catching the InterruptException has the same effect.

The interrupt flag can be (re)set to show the interrupt occurred with `Thread.currentThread().interrupt()`

Reply:
I forgot that catching `InterruptedException` clears the state. I don't understand the value of re-setting the interrupt state, I don't think that anything above this is going to check it.

Reply:
Addressed in 679612b